YAFC/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -663,7 +663,7 @@ private enum ProductDropdownType {
                         }
                     }
                 }
-                if (!allRecipes.Contains(rec) || (await MessageBox.Show("Recipe already exists", "Add a second copy?", "Add a copy", "Cancel")).choice) {
+                if (!allRecipes.Contains(rec) || (await MessageBox.Show("Recipe already exists", "Add a second copy of {rec.locName}?", "Add a copy", "Cancel")).choice) {
```

Comment:
```suggestion
                if (!allRecipes.Contains(rec) || (await MessageBox.Show("Recipe already exists", $"Add a second copy of {rec.locName}?", "Add a copy", "Cancel")).choice) {
```

Reply:
nice catch, I lost this change while (manually) applying the changes to 'rebase'!